habitat-lab/habitat/config/habitat/task/objectnav_velocity_control.yaml

```diff
@@ -0,0 +1,33 @@
+defaults:
+  - task_config_base
+  - actions:
+    - velocity_stop
```

Comment:
Why is an explicit stop action included? I thought stop was implied beneath a velocity threshold.

Reply:
We changed the logic to have an explicit stop action based on feedback from Devendra